src/vm/virtualcallstub.cpp

```diff
@@ -2085,7 +2085,8 @@ VirtualCallStubManager::Resolver(
     MethodTable * pMT, 
     DispatchToken token,
     OBJECTREF   * protectedObj, // this one can actually be NULL, consider using pMT is you don't need the object itself
-    PCODE *       ppTarget)
+    PCODE *       ppTarget,
+    BOOL          throwOnConflict)
```

Comment:
>throwOnConflict [](start = 18, length = 15)

Is this supposed to be throwOnConflict or throwOnFailure? If throwOnFailure, there is also the type equivalence case to deal with.

Reply:
Actually, I see that you handled the type equivalence case, which raises the question again on what this parameter actually means

---
In reply to: [227099736](https://github.com/dotnet/coreclr/pull/20458#discussion_r227099736) [](ancestors = 227099736)

Reply:
Should be `throwOnConflict`, like everywhere else. Fixed.